.changeset/perfect-toes-behave.md

```diff
@@ -0,0 +1,13 @@
+---
+'@sap-cloud-sdk/test-services-odata-common': minor
+'@sap-cloud-sdk/test-services-odata-v2': minor
+'@sap-cloud-sdk/test-services-odata-v4': minor
+'@sap-cloud-sdk/test-services-e2e': minor
+'@sap-cloud-sdk/generator-common': minor
+'@sap-cloud-sdk/odata-common': minor
+'@sap-cloud-sdk/generator': minor
+'@sap-cloud-sdk/odata-v2': minor
+'@sap-cloud-sdk/odata-v4': minor
+---
+
+[Fixed Issue] Add support for `useDefineForClassFields` which defaults to `true` when target is `ES2022`.
```

Comment:
[nth] Consider describing the fixed issue, that the user had (that null values were sent for properties that were not set when writing entities) instead of the technical underlying issue.